mito-ai/src/Extensions/AiChat/ChatTaskpane.tsx

```diff
@@ -1286,6 +1289,18 @@ const ChatTaskpane: React.FC<IChatTaskpaneProps> = ({
 
     const lastAIMessagesIndex = chatHistoryManager.getLastAIMessageIndex()
 
+    let processedDisplayOptimizedChatHistory: (IDisplayOptimizedChatItem | GroupedErrorMessages)[] = []
+    
+    if (agentModeEnabled) {
```

Comment:
Want to add a small comment that just explains how we handle things differently in agent vs chat mode?